### ISSUE TITLE
Bump versions on azure-devops-node-api to 10.2.2 to address CVE-2021-23358

### ISSUE THREAD

**Issue by akanieski** [microsoft/tfs-cli#380]
```console

  High            Arbitrary Code Execution                                      

  Package         underscore                                                    

  Patched in      >=1.12.1                                                      

  Dependency of   tfx-cli [dev]                                                 

  Path            tfx-cli > azure-devops-node-api > typed-rest-client >         
                  underscore                                                    

  More info       https://npmjs.com/advisories/1674                             

  High            Arbitrary Code Execution                                      

  Package         underscore                                                    

  Patched in      >=1.12.1                                                      

  Dependency of   tfx-cli [dev]                                                 


  More info       https://npmjs.com/advisories/1674                             

found 2 high severity vulnerabilities in 230 scanned packages
  2 vulnerabilities require manual review. See the full report for details.
  ```

According to [https://github.com/microsoft/azure-devops-node-api/pull/440](https://github.com/microsoft/azure-devops-node-api/pull/440) underscore was bumped up on 10.2.2.

**Comment by MOlausson** [comment]
What's the ETA to get this fixed? Also duplicate in #392.

**Comment by anatolybolshakov** [comment]
Hi everyone, let us take a look at it this week.

**Comment by max-zaytsev** [comment]
The updated package was published ([0.10.0](https://www.npmjs.com/package/tfx-cli/v/0.10.0))
